.babelrc.js

```diff
@@ -0,0 +1,13 @@
+let ignore = [`**/dist`]
+
+// Jest needs to compile this code, but generally we don't want this copied
+// to output folders
+if (process.env.NODE_ENV !== `test`) {
+  ignore.push(`**/__tests__`)
+}
+
+module.exports = {
+  sourceMaps: true,
+  presets: ["@babel/preset-env", "babel-preset-gatsby-package"],
```

Comment:
im not sure if preset-env is needed, but i didnt wanna spend any time to figure it out.